cluster/centos/config-build.sh

```diff
@@ -31,16 +31,16 @@ ETCD_VERSION=${ETCD_VERSION:-"3.0.9"}
 # Define k8s version to use.
 K8S_VERSION=${K8S_VERSION:-"1.3.7"}
 
-DOCKER_DOWNLOAD_URL=\
+export DOCKER_DOWNLOAD_URL=\
```

Comment:
These don't need exported as they are used by the same bash shell when this file is sourced....

Reply:
Ok, I’ll change and add a directive to have shellcheck ignore

Reply:
please add a comment noting that we're ignoring it for this reason :+1:

Reply:
Done!